rfcs/endpoints-needed.md

```diff
@@ -0,0 +1,50 @@
+# Enpoints Needed For Each Page
+
+Note all enpoints describe what the endpoint should do not necassarily the name of the endpoint.
+
+### Sign In Page
+
+- Get Username
+- Get Password
+- Check for match (username and password)
+- Directs to Match List Page
+
+  #### Sign Up Page  
+
+  - Check for valid creditials (unique, formatting)
+
+  #### Forgot Password Page
+
+  - Get Username 
+  - Send email with tempory creditials
+
+### Match List Page
+
+- Get all matches (filter by signed in account)
+- Create Match button (sends to create match page)
+- Match Details button (sends to match details page)
+- Get user information (logo, username, team name)
```

Comment:
I think the logo and team name will already be available when we call the `GET` all matches (since those are fields in the Fantasy Team model). 

Also, maybe there is a way to save the username from the sign in page, so we don't need to make another API request for that.